cmd/litcli/sessions.go

```diff
@@ -50,6 +50,14 @@ var addSessionCommand = cli.Command{
 			Usage: "set to true to skip verification of the " +
 				"server's tls cert.",
 		},
+		cli.StringFlag{
+			Name: "type",
+			Usage: "session type to be created which will " +
+				"determine the permissions a user has when " +
+				"connecting with the session. Options " +
+				"include readonly|admin",
+			Value: "readonly",
```

Comment:
We'll need to update our docs here as the UI assumes it has all perms (?) atm. 

I think a good slot to settle in would be normal perms w/ certain thresholds.